### ISSUE TITLE
Using $digits masculine form

### ISSUE THREAD

**Issue by DPlachkov** [kwn/number-to-words#63]
Hello.

I find this library amazing. I'm trying to convert currency to text, which for the language that i have chosen "bg" is not created, but thats ok - i can use the numbers convertor. I've checked the source code and there are 3 different forms .By default it is set to the neutral digit, but I need to use the masculine form. Is there some way to change it ?
```
 private static $digits = [
        0  => [1 => "едно", "две", "три", "четири", "пет", "шест", "седем", "осем", "девет"], // neuter
        1  => [1 => 'един', 'два'],                                                           // masculine
        -1 => [1 => 'една']                                                                   // feminine
    ];
```

The only code I use ( and this is an example ) is 
```
$numberToWords =  new \NumberToWords\NumberToWords;
$trans = $numberToWords->getNumberTransformer('bg');
$trans->toWords(2011);
```
If there is no option to toggle this, is there a way for me to extend the library and do it on my own

tldr. i need to have "един" and "два" Instead of "едно" and "две".

**Comment by kwn** [comment]
Hey @DPlachkov 

Thank you for reporting an issue. The code that you're referring to is inherited from the other library and may have some bugs. I believe the masculine, femine and neutral form were originally created for the currency translator (currencies in slavic languages have their "gender"). 

Can you provide a specific example you're trying to solve? Are you trying to transform something that needs to consider a gender form?
